flink-ml-lib/src/main/java/org/apache/flink/ml/evaluation/binaryclassification/BinaryClassificationEvaluator.java

```diff
@@ -520,13 +354,16 @@ private static long[] reduceBinarySummary(List<BinarySummary> values, int taskId
      */
     private static void updateBinarySummary(
             BinarySummary statistics, Tuple3<Double, Boolean, Double> evalElement) {
-        if (evalElement.f1) {
-            statistics.curPositive++;
+        boolean isPos = evalElement.f1;
+        double weight = evalElement.f2;
+        double score = evalElement.f0;
+        if (isPos) {
```

Comment:
The variable 'isPos' is only used once here, there is no need to declare a variable. Just use 'evalElement.f1' is OK.

Reply:
I think it's harmless to declare variable here. And it provides more readability compared to guessing meanings from the tuple.

Reply:
OK, I agree with it.